Packs/CommonScripts/ReleaseNotes/1_13_8.md

```diff
@@ -0,0 +1,10 @@
+
+#### Scripts
+
+##### New: CertificateExtract
+
+- New: Extract fields from a certificate file and return the standard context.
+
+##### New: CertificateReputation
+
+- New: Enrich and calculate the reputation of a certificate indicator. (Available from Cortex XSOAR 6.0.0).
```

Comment:
how can we determine the certificate reputation? I thought its like MITRE TTP / CVE - where you can choose the default reputation in integration config